README.md

```diff
@@ -29,6 +29,14 @@ cargo install cargo-llvm-cov
 cargo llvm-cov
 ```
 
+## Coverage Report
+
+```bash
+cargo install cargo-tarpaulin
```

Comment:
How does `cargo-tarpaulin` compare to `cargo llvm-cov`? do we need both?

Also just curious how you came to settle on `tarpaulin` / what else was considered. For example, a quick search led me to https://github.com/mozilla/grcov which seems to be what mozilla uses for firefox 🤔

Reply:
It's a great point. `cargo tarpaulin` was the first tool I found when searching for Rust coverage tools. After doing some further research, there's no need to add additional tools. We can just use `cargo llvm-cov --html` or `cargo llvm-cov --open`. `grcov` is also another option, but it requires more customized configurations.